res/Sectors/M1105/Ngathksirz.sec

```diff
@@ -18,9 +47,7 @@ Ngathksirz
 #Alleg: 13: Vb: "Vargr (Baknae Alliance)"
 #Alleg: 14: V5: "Vargr (Ruler of Five)"
 #Alleg: 15: Vd: "Vargr (Democracy of Greats)"
-#
-#
-#Created by John G. Wood <john@elvwood.org>
+
 Hex  Name                             UWP       Remarks                {Ix} (Ex)    [Cx]   N B  Z  PBG  W A      Stellar       
```

Comment:
Hmmm... this looks like it's not .SEC, but SecondSurvey. I'm kinda shocked it works... I guess if Type isn't specified the code guesses?

Can you update the PR to include a fix to the M1105 XML too?